apps/api/src/server/applications/application/__tests__/get-application-representation.test.js

```diff
@@ -0,0 +1,26 @@
+import supertest from 'supertest';
+import { app } from '../../../app.js';
+const { databaseConnector } = await import('../../../utils/database-connector.js');
+
+const request = supertest(app);
+
+const existingRepresentations = [
+	{
+		id: 1,
+		reference: 'BC0110001-2',
+		status: 'VALID',
+		redacted: true,
+		received: '2023-03-14T14:28:25.704Z'
+	}
+];
+
+describe('Get Application Representation', () => {
+	it('gets rep for a case by id', async () => {
```

Comment:
Could we add some more tests cases here, such as if the case does not exist in the db?

Reply:
addressed